module/purpose/PS3Walk/src/PS3Walk.cpp

```diff
@@ -72,62 +83,146 @@ namespace module::purpose {
+                                emit<Task>(load_script<LimbsSequence>("Crouch2.yaml"), 3);
+                            }
+                            break;
+                        case BUTTON_L1:
+                            if (event.value > 0) {
+                                log<NUClear::INFO>("Requesting Left Front Kick");
+                                emit<Task>(std::unique_ptr<Walk>(nullptr));
+                                emit<Task>(std::unique_ptr<Look>(nullptr));
+                                emit<Task>(std::make_unique<Kick>(LimbID::LEFT_LEG), 3);
+                            }
+                            break;
+                        case BUTTON_R1:
+                            if (event.value > 0) {
+                                log<NUClear::INFO>("Requesting Right Front Kick");
+                                emit<Task>(std::unique_ptr<Walk>(nullptr));
+                                emit<Task>(std::unique_ptr<Look>(nullptr));
+                                emit<Task>(std::make_unique<Kick>(LimbID::RIGHT_LEG), 3);
                             }
                             break;
-                            // case BUTTON_L1:
-                            //     if (event.value > 0) {  // button down
-                            //         NUClear::log("Requesting Left Front Kick");
-                            //         emit<Task>(std::make_unique<Kick>(LimbID::LEFT_LEG), 2);
-                            //     }
-                            //     break;
-                            // case BUTTON_R1:
-                            //     if (event.value > 0) {  // button down
-                            //         NUClear::log("Requesting Right Front Kick");
-                            //         emit<Task>(std::make_unique<Kick>(LimbID::RIGHT_LEG), 2);
-                            //     }
-                            //     break;
                             // case BUTTON_L2:
                             //     if (event.value > 0) {  // button down
                             //         NUClear::log("Requesting Left Side Kick");
+                            //         emit(std::make_unique<KickScriptCommand>(KickScriptCommand(
+                            //             -Eigen::Vector3d::UnitY(),  // vector pointing right relative to
+                            //             robot LimbID::LEFT_LEG)));
                             //     }
                             //     break;
                             // case BUTTON_R2:
                             //     if (event.value > 0) {  // button down
                             //         NUClear::log("Requesting Right Side Kick");
+                            //         emit(std::make_unique<KickScriptCommand>(KickScriptCommand{
+                            //             Eigen::Vector3d::UnitY(),  // vector pointing left relative to robot
+                            //             LimbID::RIGHT_LEG}));
```

Comment:
I'd get rid of all this, the code won't compile anyway. It might be good to keep the cases in, but have them not do anything or just put a log in.